Riot/Categories/UINavigationController+Riot.m

```diff
@@ -49,4 +49,14 @@ - (UIInterfaceOrientation)preferredInterfaceOrientationForPresentation
     return UIInterfaceOrientationUnknown;
 }
 
+- (UIViewController *)childViewControllerForStatusBarStyle
```

Comment:
We should use `RiotNavigationController` instead where needed. In fact we should not override existing method in categories as said in Objective-C  [documentation](https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/ProgrammingWithObjectiveC/CustomizingExistingClasses/CustomizingExistingClasses.html#//apple_ref/doc/uid/TP40011210-CH6-SW4) this can lead to undefined behavior.

Reply:
Well, actually all these seems redundant for now. Works same without them. Deleted them all.

Reply:
Changed all navigation controllers to RiotNavigationController's.